x-pack/solutions/security/plugins/security_solution/public/common/lib/kuery/index.ts

```diff
@@ -225,10 +225,11 @@ export const convertToBuildEsQuery = ({
   filters: Filter[];
 }): [string, undefined] | [undefined, Error] => {
   try {
+    const newDataViewExists = dataView?.id && dataView?.title;
```

Comment:
Any reason for using `title` here which is deprecated? Why not using `getIndexPattern()`?
Also maybe even better, we could use this [hasMatchedIndices](https://github.com/elastic/kibana/blob/main/src/platform/plugins/shared/data_views/common/data_views/abstract_data_views.ts#L568) function?

Reply:
I'll change it, but the definition for `getIndexPattern()` is 

```
  getIndexPattern = () => this.title;
```

Reply:
> I'll change it, but the definition for getIndexPattern() is getIndexPattern = () => this.title;

Yup yup I know, it's just if someone wants to remove the export of `this.title` then it's one less place to change 😉 
Thanks for updating it!